hls4ml/writer/vitis_writer.py

```diff
@@ -0,0 +1,31 @@
+import os
+import glob
+from shutil import copy
+from hls4ml.writer.vivado_writer import VivadoWriter
+
+class VitisWriter(VivadoWriter):
+
+    def __init__(self):
+        super().__init__()
+
+    def write_nnet_utils_overrides(self, model):
+        ###################
```

Comment:
The comment should be made clearer (and changed to a docstring)

Reply:
Really? Pretty sure it is obvious to you and everyone who looked what this does. People who can't figure this out shouldn't be looking at the codebase in the first place :smile:. Jokes aside, like mentioned in the other comment, docstrings will be addressed in a separate PR.